src/main/java/com/blackducksoftware/integration/hub/alert/web/SSLAuthenticationHandler.java

```diff
@@ -45,12 +54,26 @@ protected void configure(final HttpSecurity http) throws Exception {
                 "/js/bundle.js.map",
                 "/css/style.css",
                 "index.html",
-                "/api/configuration/provider/hub",
                 "/api/login",
                 "/api/logout" };
-        http.requiresChannel().anyRequest().requiresSecure().and().csrf().disable().authorizeRequests().antMatchers(allowedPaths).permitAll().and()
+
+        final String[] csrfIgnoredPaths = {
+                "/",
+                "/#",
+                "/favicon.ico",
+                "/fonts/**",
+                "/js/bundle.js",
+                "/js/bundle.js.map",
+                "/css/style.css",
+                "index.html",
+                "/api/login" };
+        http.requiresChannel().anyRequest().requiresSecure().and().csrf().csrfTokenRepository(csrfTokenRepository).ignoringAntMatchers(csrfIgnoredPaths).and().authorizeRequests().antMatchers(allowedPaths).permitAll().and()
                 .authorizeRequests().anyRequest().hasRole("ADMIN").and()
```

Comment:
Given our discussion today about chaining long methods like this, do we want to modify this so that it's easier to read and debug? Or is this ok to have since it's not as complicated as the one we reviewed earlier?